docs/reference/triggers/kafka.md

```diff
@@ -55,9 +55,9 @@ A large partition consumer queue will reduce processing delays (as there will al
 The Nuclio trigger reads directly from this partition consumer queue (remember - there is one such message queue per partition) and for each message allocates a worker and sends the message to be handled. When the handler returns, a new message is read from the queue and handled.
 
 ### Configuration parameters
-* Fetch Min (`trigger.attributes.fetchMin`, `nuclio.io/kafka-fetch-min`): The minimum number of message bytes to fetch in a request - the broker will wait until at least this many are available. The default is 1, as 0 causes the consumer to spin when no messages are available. Equivalent to the JVM's `fetch.min.bytes`.
-* Fetch Default (`trigger.attributes.fetchDefault`, `nuclio.io/kafka-fetch-default`): The default number of message bytes to fetch from the broker in each request (default 1MB). This should be larger than the majority of your messages, or else the consumer will spend a lot of time negotiating sizes and not actually consuming. Similar to the JVM's `fetch.message.max.bytes`.
-* Fetch Max (`trigger.attributes.fetchMax`, `nuclio.io/kafka-fetch-max`): The maximum number of message bytes to fetch from the broker in a single request. Messages larger than this will return ErrMessageTooLarge and will not be consumable, so you must be sure this is at least as large as your largest message. Defaults to 0 (no limit). Similar to the JVM's `fetch.message.max.bytes`. The global `sarama.MaxResponseSize` still applies.
+* Fetch Min (`trigger.attributes.fetchMin`, `nuclio.io/kafka-fetch-min`): The minimum number of message bytes to fetch in a request - the broker will wait until at least this many are available. The default is 1, as 0 causes the consumer to spin when no messages are available. Equivalent to the JVMs `fetch.min.bytes`.
+* Fetch Default (`trigger.attributes.fetchDefault`, `nuclio.io/kafka-fetch-default`): The default number of message bytes to fetch from the broker in each request (default 1MB). This should be larger than the majority of your messages, or else the consumer will spend a lot of time negotiating sizes and not actually consuming. Similar to the JVMs `fetch.message.max.bytes`.
+* Fetch Max (`trigger.attributes.fetchMax`, `nuclio.io/kafka-fetch-max`): The maximum number of message bytes to fetch from the broker in a single request. Messages larger than this will return ErrMessageTooLarge and will not be consumable, so you must be sure this is at least as large as your largest message. Defaults to 0 (no limit). Similar to the JVMs `fetch.message.max.bytes`. The global `sarama.MaxResponseSize` still applies.
```

Comment:
It seems to me the meaning was possession: "of the JVM", and not to pluralize JVM. 🤔

Reply:
Fixing ...